### DISCUSSION TITLE
Does parcel v2 support autoinstall of imported packages?

### DISCUSSION THREAD

**Discussion by himanshugarg** [parcel-bundler/parcel#7632] [Q&A] [answered]
Hi, when trying [this example with parcel (not parcel-bundler) on mdn](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Package_management), it appeared that autoinstall of packages was no longer available. The autoinstall works fine with parcel-bundler. I did check the [migration docs ](https://parceljs.org/getting-started/migration/) but couldn't find specific mentions about this. The change, if any, is not a problem. I just want to [confirm I am not missing something](https://github.com/mdn/content/issues/11984#issuecomment-1025080599). Thanks

**Comment by mischnic** [comment, marked as answer]
No, it doesn't. (And this was a security problem anyway)
